packages/enhanced/src/lib/container/ModuleFederationPlugin.ts

```diff
@@ -77,6 +80,14 @@ class ModuleFederationPlugin implements WebpackPluginInstance {
         compiler,
       );
     }
+    if (options.embedRuntime) {
+      //@ts-ignore
+      new StartupChunkDependenciesPlugin({
```

Comment:
if true, will use require.X and custom startup runtime module for entrypoints. 
Removes need for import()